### PR TITLE
Homework1: Maryia Lichko

### PR DESCRIPTION
**Pull request by maryiaLichko**
Maryia Lichko

### CODE REVIEW

**Comment by maryiaLichko** [comment]
Homework 1,
 name: Maryia Lichko

**Review by belovmd — CHANGES_REQUESTED** [review]
No changes please fix

File: homework1/my homework #1 [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (belovmd) [inline homework1/my homework #1]: why do we need empty file?

**Review by belovmd — COMMENTED** [review]
1. please remove second homework files
2. currently after issue-1 fix it's smth like +0,5 points. to improve it up to +2 please add additional tasks

**Comment by maryiaLichko** [comment]
> * please remove second homework files
> * currently after issue-1 fix it's smth like +0,5 points. to improve it up to +2 please add additional tasks

Tasks are added (2nd is in src folder*)
why should I remove 2nd homework?

**Comment by belovmd** [comment]
> > * please remove second homework files
> > * currently after issue-1 fix it's smth like +0,5 points. to improve it up to +2 please add additional tasks
> 
> Tasks are added (2nd is in src folder*)
> why should I remove 2nd homework?

Because it's 1st homework
1st in Homework1
2nd in Homework2
etc. 
No need keep all code in one PR - it's inaccurate and difficult to review

**Comment by maryiaLichko** [comment]
> > > * please remove second homework files
> > > * currently after issue-1 fix it's smth like +0,5 points. to improve it up to +2 please add additional tasks
> > 
> > 
> > Tasks are added (2nd is in src folder*)
> > why should I remove 2nd homework?
> 
> Because it's 1st homework
> 1st in Homework1
> 2nd in Homework2
> etc.
> No need keep all code in one PR - it's inaccurate and difficult to review

checks  done , please review

**Review by belovmd — COMMENTED** [review]
+1,5
please next time put tasks descriptions to files (it hasn't affected this HW)

**Pull request closed without merging**